pvlib/pvsystem.py

```diff
@@ -2648,13 +2648,16 @@ def scale_voltage_current_power(data, voltage=1, current=1):
 
     # as written, only works with a DataFrame
     # could make it work with a dict, but it would be more verbose
+    voltage_keys = ['v_mp', 'v_oc']
+    current_keys = ['i_mp', 'i_x', 'i_xx', 'i_sc']
+    power_keys = ['p_mp']
     data = data.copy()
-    voltages = ['v_mp', 'v_oc']
-    currents = ['i_mp', 'i_x', 'i_xx', 'i_sc']
+    voltages = voltage_keys and data.columns
+    currents = current_keys and data.columns
+    powers = power_keys and data.columns
```

Comment:
I was thinking of something like this:

```python
In [28]: voltage = 5

In [29]: current = 10

In [30]: df = pd.DataFrame([[1, 1, 1]], columns=['p_mp', 'v_mp', 'i_mp'])

In [31]: df
Out[31]:
   p_mp  v_mp  i_mp
0     1     1     1

In [32]:     voltage_keys = ['v_mp', 'v_oc']
    ...:     current_keys = ['i_mp', 'i_x', 'i_xx', 'i_sc']
    ...:     power_keys = ['p_mp']

In [33]: df.filter(voltage_keys, axis=1) * voltage
Out[33]:
   v_mp
0     5

In [34]: df.filter(current_keys, axis=1) * current
Out[34]:
   i_mp
0    10
```

Reply:
```
In [35]: voltage_df = df.filter(voltage_keys, axis=1) * voltage

In [36]: current_df = df.filter(current_keys, axis=1) * current

In [37]: df = pd.concat([voltage_df, current_df], axis=1)

In [38]: df['p_mp'] = df['v_mp'] * df['i_mp']

In [39]: df
Out[39]:
   v_mp  i_mp  p_mp
0     5    10    50
```

Reply:
I don't know how to scale anything other than `p_mp` without hugely complicating the code. I also don't know that there's a need for it at this time.

Reply:
The `pvwatts` methods don't produce voltage or current.

Reply:
I was still in the mindset that we could provide a DataFrame with `p_mp=pdc`, `v_mp=pdc`, and `i_mp=1` as I suggested above. But yes it is better to just multiply the power keys by the voltage and current scale factors.

Reply:
One issue with the temporary DataFrame: when converting back to Series, the column label `'p_mp'` is assigned to the Series name, which affects testing but likely no user code. Any concern here?

Reply:
no concern. pandas `assert_series_equal` only recently started checking that attribute.